README.md

```diff
@@ -1,2 +1,18 @@
 # census-eq-monitoring
-Terraform scripts for provisioning Census EQ Stackdriver Monitoring Policies to Google Cloud
+This repository contains terraform scripts for provisioning Census EQ Stackdriver Monitoring and Alert Policies to Google Cloud Platform.
+
+## Prerequisites
+Provision an environment following the instructions found in the census-eq-terraform [README](https://github.com/ONSdigital/census-eq-terraform/edit/master/README.md), taking care to apply those that relate to Stackdriver Workspaces.
```

Comment:
The Stackdriver workspaces section was removed from census-eq-terraform. We should add it here so that people know how to set one up.